crates/sui-framework/sources/object.move

```diff
@@ -39,6 +39,12 @@ module sui::object {
         id: ID,
     }
 
+    // === address ===
```

Comment:
nit, we should add `EAddressParseError` to this file, even if not used, since it helps keep the abort codes separate

Reply:
Will do!  I was also thinking about this more generally -- that the current strategy of counting up from zero in each module is probably not great when you account for aborts coming from nested stackframes, and that at least for the framework we could probably have a more structured approach (e.g. partition the bits in the error code into module, function and code).